components/Collections/src/private/MDCCollectionInfoBarView.m

```diff
@@ -232,17 +232,17 @@ - (void)dismissAnimated:(BOOL)animated {
       delay:0
       options:UIViewAnimationOptionCurveEaseIn
       animations:^{
-        _backgroundView.transform = CGAffineTransformMakeTranslation(0, _backgroundTransformY);
+        self.backgroundView.transform = CGAffineTransformMakeTranslation(0, self->_backgroundTransformY);
```

Comment:
Nit/Formatting: This looks like it's over 100 columns. Please break it into 2 lines.

Reply:
done.